src/components/Icon/index.tsx

```diff
@@ -0,0 +1,31 @@
+import styled from "styled-components";
+
+import { primaryText } from "../../assets/styles/colors";
+import { minDevice } from "../../assets/styles/breakpoints";
+
+interface IconProps {
+  fontSize: string;
+  color: string;
+  width: string;
+  hoverColor: string;
+  paddingRight: string;
+}
+
+type PropsOptional = Partial<IconProps>;
+
+const Icon = styled.div<PropsOptional>`
+  color: ${(props) => props.color || primaryText};
```

Comment:
Meta thought - are you used to have arrow parens always? The thought behind that rule is described here:
```
Always include parentheses around arguments for clarity and consistency. eslint: arrow-parens
Why? Minimizes diff churn when adding or removing arguments.
```
https://github.com/airbnb/javascript#arrows--one-arg-parens